assets/tests/consolidated_1_1_tests.yaml

```diff
@@ -2787,6 +3159,17 @@ tests:
                 relation: parent
                 user: user:* #invalid
             errorCode: 2027
+        listUsersAssertions:
+          - request:
+              filters:
+                - user
+              object: document:1
+              relation: viewer
+            contextualTuples:
+              - object: document:1
+                relation: parent
+                user: user:* #invalid
```

Comment:
Add another test elsewhere where `user: group#undefined` for example. I don't see a test where we pass in a contextual tuple to ListUsers that defines a user that has an undefined relation on the userset relation.

I may have overlooked a test though..

Reply:
added: `validation_invalid_userset_in_contextual_tuple`